modules/planning/reference_line/reference_line_provider.cc

```diff
@@ -317,8 +317,9 @@ bool ReferenceLineProvider::GetReferenceLinesFromRelativeMap(
     return false;
   }
   // get curent adc lane info by vehicle state
-  common::VehicleState vehicle_state =
-      common::VehicleStateProvider::instance()->vehicle_state();
+  // since the relative map is based on the coordinate system of the car itself,
+  // the vehicle is the origin of the coordinate system
+  common::VehicleState vehicle_state;
```

Comment:
always get vehicle_state from vehicle state provider, to centralize the vehicle state generation logic.